src/test/ui/consts/miri_unleashed/abi-mismatch.stderr

```diff
@@ -17,7 +17,7 @@ LL |     my_fn();
    |     ^^^^^^^
    |     |
    |     calling a function with ABI C using caller ABI Rust
-   |     inside call to `call_rust_fn` at $DIR/abi-mismatch.rs:13:17
+   |     inside `VAL` at $DIR/abi-mismatch.rs:13:17
```

Comment:
we are inside `call_rust_fn`, so the previous message mentioning the current function made sense to me. Now it sounds like we are calling `my_fn` inside `VAL`, which we are not. It feels to me like the info about the fact that the current function is called from `VAL` is missing

Reply:
This is probably caused by skipping the first frame. I can make it not do that. Then the irignal source location of the error will be duplicated though.

Reply:
I have the patch done and the output is right but I cannot figure out how to write the `ERROR` annotations in the rs file to make it accept the output... :/